src/index.js

```diff
@@ -61,11 +64,20 @@ const formatOptions = ({steps, ...otherOptions}) => ({
   steps: (steps || ['welcome','document','face','complete']).map(formatStep)
 })
 
+const setI18n = (options) => {
```

Comment:
any reason for calling it `i18n` if we are not using `i18n`? is it a standard term on js?

Reply:
`i18n` stands for `internationalization` and this function is just setting the internationalization. Therefore I've decided to use `i18n` independently of the library we use as it's more generic and representative of what the function does.